docs/userGuide/usingComponents.md

```diff
@@ -1,9 +1,10 @@
-<variable name="title" id="title">Using Components</variable>
```

Comment:
> In every page? Or just in the layout pages?

Just layouts : )

Still something extra to take note of for the author though

> > * a heavy shift of layout related styles to userland - e.g. `#content-wrapper`, `#site-nav`, .... This now becomes the user's responsibility to maintain. (https://github.com/MarkBind/markbind/pull/1403/files#diff-309b964023ca899c9505205f36d3f4d5b36a6487e5c9b2e242204ee06bbc9ce9)
> 
> I noticed `<span id="title" class="d-none">{{ title }}</span>` being added to many pages. Is it quick of your own site or something authors have to do in every page in every site?

This one would be belong to the page `<variable>` / `<import>` removal pr (to be split), there's no way to set the id for a nunjucks tag so had to create another `<span>`. The title `id` is used for the userguide macro that creates the left / right navigation buttons at the bottom

Reply:
Got it.